cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -776,13 +786,28 @@ def test_load_values_on_init_option_service_dialog_element():
         casecomponent: Services
         initialEstimate: 1/16h
         testSteps:
-            1. create a service dialog
+            1. create a service dialog with option dynamic to True
             2. In service dialog element option page
         expectedResults:
             1.
             2. Load_values_on_init button should always be enabled
     """
-    pass
+    element_data = {
+        'element_information': {
+            'ele_label': fauxfactory.gen_alphanumeric(15, start="ele_label_"),
+            'ele_name': fauxfactory.gen_alphanumeric(15, start="ele_name_"),
+            'ele_desc': fauxfactory.gen_alphanumeric(15, start="ele_desc_"),
+            'choose_type': "Text Box",
+            'dynamic_chkbox': True
+        }
+    }
+    dialog, element = create_dialog(appliance, element_data)
+    view = appliance.browser.create_view(EditElementView)
+    view.element.edit_element(element_data['element_information']['ele_label'])
+    assert view.element_information.dynamic_chkbox.is_enabled
+    view.options.click()
+    request.addfinalizer(dialog.delete_if_exists)
```

Comment:
Best practice: Always put finalizer after step of creation. here I think your dialog creating at L804.